package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pagopa-selfcare-backoffice-frontend",
-  "version": "1.41.7",
+  "version": "1.41.7-2-CHK-4094-refactor-backoffice-client-api",
```

Comment:
Is this dirty reference wanted?

Reply:
this is not a _dirty reference_ but is a patch performed by deployment pipeline in various envs (see committer [here](https://github.com/pagopa/pagopa-selfcare-frontend/pull/810/commits/408c3bcd58f7780d83a18b73932a1ee7ba66f3cf)) from deploy pipeline, will be overridden once deployed from main :)